src/asciidoc/docs/develop/portExistingCode.adoc

```diff
@@ -4,354 +4,149 @@
-. Open the `frontend/index.js` file, and add the following code below the `upgradeBGColor` function.
-+
-This code sets the initial sounds based on the customization files and adds a function for dynamically loading sounds.
-+
-[source,javascript]
+npm install
+npm start
 ----
-// Set the initial sounds
-let laserSound = instantRemixing.get(['settings', 'gameOptions', 'laserSound']);
-let explosionSound = instantRemixing.get(['settings', 'gameOptions', 'explosionSound']);
+. Refresh the preview to see the Mortgage Overpayment Calculator.
 
-const loadAudio = () => {
-  SFX = {
-    laser: new Audio(laserSound),
-    explosion: new Audio(explosionSound),
-  };
-
-  // preload audio
-  for (var sfx in SFX) {
-    (function () {
-      var audio = SFX[sfx];
-      SFX[sfx] = function () {
-        return audio.play();
-      }
-    })();
-  }
-};
-----
-. Update the `onValueChanged` listener to monitor changes to the sound values.
-+
-[source,javascript]
-----
-// Listen for updates
-instantRemixing.onValueChanged((path, newValue) => {
-  if (path[0] && path[0] === 'settings' && path[1] && path[1] === 'gameOptions') {
-    laserSound = newValue.laserSound;
-    explosionSound = newValue.explosionSound;
-    upgradeBGColor(newValue.backgroundColor);
-    loadAudio();
-  }
-});
-----
-. Press *Ctrl+F* to open the finder tool, and enter `.wav` to search for the WAV files and audio preloading that the game was using.
-+
-It should look like the following example.
-+
-[source,javascript]
-----
-SFX = {
-  laser:     new Audio('39459__THE_bizniss__laser.wav'),
-  explosion: new Audio('51467__smcameron__missile_explosion.wav')
-};
+== Publishing your template
 
-// preload audio
-for (var sfx in SFX) {
-  (function () {
-    var audio = SFX[sfx];
-    SFX[sfx] = function () {
-      return audio.play();
-    }
-  })();
-}
-----
+You are now ready to test a live build of your project.
 
-. Replace the code you found with a call to your new `loadAudio` function.
+. Click *Publish Now* on the left side of the editor.
 +
-[source,javascript]
-----
-loadAudio();
-----
-. Press *Ctrl+S* to save the file.
-+
-The template now references the user-selected values for those sounds.
-You can return to the *App settings* configuration, and use the sound pickers to replace these sounds with whatever you want.
-You can even try just recording some sounds yourself.
-For example, you can try "pew" for the laser and "pow!" for the explosion.
+You will be prompted to enter additional information about your template, such as the name and description.
+. Click *Publish New Version*.
 +
-Changes to the sound choices should trigger a refresh of your template, and you should be able to hear the new sounds.
-. Publish your changes to release an updated version of your template.
+Publishing runs the build commands specified for the project, and then displays a live link to your production template.
 
 == Wrapping up
 
```

Comment:
The tutorial no longer shows how to "integrate with Koji awesomeness." It might be better to include the next steps after publishing the app. Namely, to make it configurable by creators (for example, allow them to change the name or the color scheme). Then, add it to the App Store so that creators can add it to their profile.